### ISSUE TITLE
Update examples to use Keras where possible

### ISSUE THREAD

**Issue by mortendahl** [tf-encrypted/tf-encrypted#416]

**Comment by jvmancuso** [comment]
We should consider waiting on this until we finish refactoring the converter, unless we can do it with the current converter (related https://github.com/mortendahl/tf-encrypted/issues/297)

**Comment by mortendahl** [comment]
Not sure I follow. Most of the examples are not using the converter.

**Comment by ianlivingstone** [comment]
Duplicate of #541
